bundler/lib/bundler/plugin/installer.rb

```diff
@@ -77,7 +77,7 @@ def install_git(names, version, options)
 
       def install_path(names, version, path)
         source_list = SourceList.new
-        source = source_list.add_path_source({ "path" => path })
+        source = source_list.add_path_source({ "path" => path, "root_path" => Pathname.new(Dir.pwd) })
```

Comment:
I think

```suggestion
        source = source_list.add_path_source({ "path" => path, "root_path" => SharedHelpers.pwd })
```
gives you a Pathname already.

Reply:
Done.